cmd/cape/cmd/root.go

```diff
@@ -145,6 +145,12 @@ func initConfig() {
 	}
 	viper.SetDefault("ENCLAVE_HOST", "wss://enclave.capeprivacy.com")
 
+	if err := viper.BindEnv("SUPERVISOR_HOST"); err != nil {
+		log.Error("failed to bind config variable.")
+		cobra.CheckErr(err)
+	}
+	viper.SetDefault("SUPERVISOR_HOST", "app.capeprivacy.com")
```

Comment:
for consistency (see line 140), can we update this to `https://app.capeprivacy.com`